roles/pulp_devel/tasks/main.yml

```diff
@@ -45,10 +45,64 @@
       become: true
       become_user: "{{ developer_user }}"
 
-- import_tasks: config_files.yml
```

Comment:
So for this entire block, you should just leave them inside config_files.yaml.

The only task you'd add to this file is your new task.

Reply:
I'll do it

Reply:
Done